.travis.yml

```diff
@@ -1,37 +1,83 @@
-#if: type = pull_request
+if: type = pull_request
+
+jobs:
+  include:
+
+  - stage: test
+    before_script:
+    - echo $DOCKER_TOKEN | docker login --username $DOCKER_USERNAME --password-stdin
+    - docker pull samseedsoftwire/todo-app-test:latest
+    - docker build --target test --tag samseedsoftwire/todo-app-test:latest .
+    script:
+    - docker run samseedsoftwire/todo-app-test:latest tests_unit -rA
+    - docker run samseedsoftwire/todo-app-test:latest tests_integration -rA
+    - docker run -e AUTH_PARAMS_KEY=$AUTH_PARAMS_KEY -e AUTH_PARAMS_TOKEN=$AUTH_PARAMS_TOKEN -e BOARD_ID=$BOARD_ID -e FLASK_APP=$FLASK_APP -e FLASK_ENV=$FLASK_ENV samseedsoftwire/todo-app-test:latest tests_e2e -rA
```

Comment:
There's a slightly simpler syntax we can take advantage of here, each env  variable if it's not changing names can be passed through directly like
`... -e AUTH_PARAMS_KEY -e AUTH_PARAMS_TOKEN..` etc.